### ISSUE TITLE
Difficulty Understanding the backpropagation part of your code

### ISSUE THREAD

**Issue by ukdsvl** [GT-RIPL/FeatMatch#1]
Hi, Can you please highlight the part of the code that performs backpropagation, and updation of the model (via         loss.backward(), opt.step() ) ?

**Comment by ukdsvl** [comment]
closing as issue is resolved
